README.md

```diff
@@ -205,6 +205,24 @@ Usage: cewl [OPTIONS] ... <url>
     <url>: The site to spider.
 ```
 
+### Running CeWL in a Docker container
+
+To quickly use CeWL on your machine with Docker, you have to build it :
+1. Build the container :
+    ```sh
+    sudo docker build -t cewl .
+    ```
+2. Container usage without interacting with local files :
+    ```sh
+    sudo docker run -it --rm cewl [OPTIONS] ... <url>
+    ```
+3. Container usage with local files as input or output :
+    ```sh
+    # you have to mount the current directory when calling the container 
+    sudo docker run -it --rm -v "${PWD}:/host" cewl [OPTIONS] ... <url>
+    ```
```

Comment:
I'd prefer the use of `sudo` to be optional. We should not be running docker images as root.
Perhaps remove sudo and put a comment at the beginning or the end like:

> Depending on your host configuration you may need to run the docker commands using `sudo`

Reply:
Done :)